.gitignore

```diff
@@ -1,12 +1,13 @@
-**/.DS_Store
-node_modules
+.DS_Store
+node_modules/
 lib
 umd
-dist
+dist/
 .vscode
 .yarn/*
 !.yarn/releases
 !.yarn/plugins
-.next
+.next/
+.idea/
```

Comment:
Wondering why'd you add the trailing slash?

Reply:
To explicitly mention that it’s a folder, otherwise, git could ignore `.idea` file, that will never happen in our case, but it helps to dev to understand that we ignore folder and not file

Reply:
Ok, good point. Then we should add the same to the rest?

Reply:
Yeah, you can add, I just added to things where I am sure that these are the files 😅